modules/operator-deploy-infrastructure.adoc

```diff
@@ -97,42 +97,92 @@ Use the following procedure to create a project with node selector and toleratio
 
 [NOTE]
 ====
-If you have already deployed {productname} using the Operator, remove the installed Operator and any specific namespaces that you created for the deployment. 
+The following procedure can also be completed by removing the installed {productname} Operator and the namespace, or namespaces, used when creating the deployment. Users can then create a new resource with the following annotation.
 ====
 
 .Procedure
 
-. Create a project resource, specifying a node selector and toleration. For example:
+. Enter the following command to edit the namespace where {productname} is deployed, and the following annotation: 
 +
-.quay-registry.yaml
 [source,terminal]
 ----
-kind: Project
-apiVersion: project.openshift.io/v1
-metadata:
-  name: quay-registry
-  annotations:
-    openshift.io/node-selector: 'node-role.kubernetes.io/infra='
-    scheduler.alpha.kubernetes.io/defaultTolerations: >-
-      [{"operator": "Exists", "effect": "NoSchedule", "key":
-      "node-role.kubernetes.io/infra"}]
+$ oc annotate namespace <namespace> openshift.io/node-selector='node-role.kubernetes.io/infra='
```

Comment:
Actually here need two annotations, you can use **"oc edit namespace <quay_namespace>"** to edit, or using **"oc annotate namespace <annotation>"** command to add required annotations.

openshift.io/node-selector: 'node-role.kubernetes.io/infra='
scheduler.alpha.kubernetes.io/defaultTolerations: >-
       [{"operator": "Exists", "effect": "NoSchedule", "key":
       "node-role.kubernetes.io/infra"}]

Reply:
@LiZhang19817 Sorry, a little confused because of how it rendered. We are suggestion the following command, right? 

```suggestion
$ oc annotate namespace <namespace> openshift.io/node-selector='node-role.kubernetes.io/infra=' scheduler.alpha.kubernetes.io/defaultTolerations='key=value:NoSchedule'
```

Thanks. Apologies!